Cargo.toml

```diff
@@ -8,16 +8,16 @@ license = "MIT"
 [dependencies]
 byteorder = "1.3.2"
 cfg-if = "1.0"
-memoffset = "0.5.1"
+memoffset = "0.6"
 minidump-common = "0.10"
 scroll = "0.11"
 tempfile = "3.1.0"
 thiserror = "1.0.21"
 
 [target.'cfg(unix)'.dependencies]
 libc = "0.2.74"
-goblin = "0.1.2"
-memmap2 = "0.2.2"
+goblin = "0.5"
+memmap2 = "0.5"
```

Comment:
Do you need to bump dependencies for later PRs? In this one it doesn't seem to be relevant. I usually try to avoid it, because vendoring in Firefox causes additional overhead. But I'll ask them how big of a deal it is. Maybe they are fine with bumping.

Reply:
Ok, bumping is fine.